src/xrt.jl

```diff
@@ -240,6 +240,8 @@ struct XRTArray{T, Dims, N} <: AbstractArray{T, N}
     end
 end
 
+Base.collect(xs::XRTArray) = xs.storage.localstorage
```

Comment:
Should be `convert(Array, xs)` to make sure to get the data from the remote if not present.